config/routes.rb

```diff
@@ -17,8 +17,13 @@
 
     resource :education_benefits_claims, only: [:create]
 
+    resource :health_care_application, only: [:create] do
+      resource :healthcheck, only: [:show], to: 'health_care_applications#healthcheck'
+    end
+
     resource :disability_rating, only: [:show]
-    resources :disability_claims, only: [:index, :show] do
+    # TODO: How to handle old URLs? Redirect?
```

Comment:
The following will point both routes to the same controller, and then we can remove the old routes once the FE is updated
```ruby
# TODO: Remove this resource/subresource when FE is updated
resources :disability_claims, only: [:index, :show], controller: 'evss_claims', as: :evss_claim do
      post :request_decision, on: :member
      resources :documents, only: [:create]
    end
```